src/AxisSets.jl

```diff
@@ -70,6 +70,21 @@ const DEFAULT_PROD_DELIM = :ᵡ
 # Short hand type for complicated union of nested Keyed or NamedDims arrays
 const XArray{L, T, N} = Union{NamedDimsArray{L,T,N,<:KeyedArray}, KeyedArray{T,N,<:NamedDimsArray}}
 
+# There's a few places calling `only` is convenient, even for older Julia releases
+if VERSION < v"1.4"
```

Comment:
Not sure if we wanna bother unit testing this given that's just around so we don't need to drop Jula 1.0 or 1.3 support.